src/components/Filters/Filters.tsx

```diff
@@ -212,6 +215,12 @@ class Filters extends React.Component<ComposedProps, State> {
       );
     });
 
+    const appliedFiltersCount = appliedFilters ? appliedFilters.length : 0;
+    const filterCountMarkup =
+      suppressTagOutput && appliedFiltersCount > 0
+        ? ` (${appliedFiltersCount})`
```

Comment:
This needs to be translated I think. IIRC some languages display counts before the word and parentheses become another character as well.